kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -499,6 +501,63 @@ object KyuubiConf {
     .checkValue(p => p == 0 || (p > 1024 && p < 65535), "Invalid Port number")
     .createWithDefault(10099)
 
+  val FRONTEND_MYSQL_BIND_HOST: OptionalConfigEntry[String] = buildConf("frontend.mysql.bind.host")
+    .doc("Hostname or IP of the machine on which to run the MySQL frontend service.")
+    .version("1.4.0")
+    .stringConf
+    .createOptional
+
+  val FRONTEND_MYSQL_BIND_PORT: ConfigEntry[Int] = buildConf("frontend.mysql.bind.port")
+    .doc("Port of the machine on which to run the MySQL frontend service.")
+    .version("1.4.0")
+    .intConf
+    .checkValue(p => p == 0 || (p > 1024 && p < 65535), "Invalid Port number")
+    .createWithDefault(3309)
+
+  val FRONTEND_MYSQL_NETTY_BOSS_THREADS: ConfigEntry[Int] =
+    buildConf("frontend.mysql.netty.boss.threads")
+      .doc("Thread number in the netty boss event loop of MySQL frontend service")
+      .version("1.4.0")
+      .intConf
+      .createWithDefault(1)
+
+  val FRONTEND_MYSQL_NETTY_WORKER_THREADS: ConfigEntry[Int] =
+    buildConf("frontend.mysql.netty.worker.threads")
+      .doc("Number of thread in the netty worker event loop of MySQL frontend service")
+      .version("1.4.0")
+      .intConf
+      .createWithDefault(4)
```

Comment:
option

Reply:
It's reasonable to use the `DEFAULT_EVENT_LOOP_THREADS` for worker event loop, but it is not suitable for Boss event loop.
```
DEFAULT_EVENT_LOOP_THREADS = Math.max(1, SystemPropertyUtil.getInt(
                "io.netty.eventLoopThreads", NettyRuntime.availableProcessors() * 2));
```